docker/ubuntu_20_04_lofar

```diff
@@ -23,8 +23,7 @@ RUN mkdir /lofarbeam/build && cd /lofarbeam/build && cmake -DCMAKE_INSTALL_PREFI
 ADD . /src
 WORKDIR /src
 
-RUN mkdir /build && cd /build && cmake ../src
-RUN cd /build && make -j2
-RUN cd /build && make install -j2
+RUN mkdir /build && cd /build && cmake -DCMAKE_CXX_FLAGS_CUSTOM=-Werror ../src
```

Comment:
For now, let's not enable error on Warnings in CI. If we find this leaves us with warnings that are a problem, we can enable it.

Reply:
While we are working here, would it be an idea to change the 4 run commands to a single run command. Docker will create a layer for each RUN command resulting in a larger container, which feels unnecessary here.

Reply:
@lkrombeenc fine to me, can you commit that change to this branch? (assuming @mnijhuis-tos is finished with the branch)

Reply:
Note: This branch has no changes in the docker files anymore. I don't mind having extra improvements in this branch but you can also create a new branch.

Reply:
Right, a new branch is probably better.